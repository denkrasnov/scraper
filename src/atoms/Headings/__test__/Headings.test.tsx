import React from "react";
import { shallow } from "enzyme";

import { H1, H2, H3, H4, H5, H6 } from "..";

describe("Headings", () => {
  const children = "__CHILDREN__";

  it("renders H1", () => {
    const component = shallow(<H1>{children}</H1>);
    expect(component).toMatchSnapshot();
  });

  it("renders H2", () => {
    const component = shallow(<H2>{children}</H2>);
    expect(component).toMatchSnapshot();
  });

  it("renders H3", () => {
    const component = shallow(<H3>{children}</H3>);
    expect(component).toMatchSnapshot();
  });

  it("renders H4", () => {
    const component = shallow(<H4>{children}</H4>);
    expect(component).toMatchSnapshot();
  });

  it("renders H5", () => {
    const component = shallow(<H5>{children}</H5>);
    expect(component).toMatchSnapshot();
  });

  it("renders H6", () => {
    const component = shallow(<H6>{children}</H6>);
    expect(component).toMatchSnapshot();
  });
});
