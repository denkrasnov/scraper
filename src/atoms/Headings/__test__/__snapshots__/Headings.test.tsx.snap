// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Headings renders H1 1`] = `
<h1>
  __CHILDREN__
</h1>
`;

exports[`Headings renders H2 1`] = `
<h2>
  __CHILDREN__
</h2>
`;

exports[`Headings renders H3 1`] = `
<h3>
  __CHILDREN__
</h3>
`;

exports[`Headings renders H4 1`] = `
<h4>
  __CHILDREN__
</h4>
`;

exports[`Headings renders H5 1`] = `
<h5>
  __CHILDREN__
</h5>
`;

exports[`Headings renders H6 1`] = `
<h6>
  __CHILDREN__
</h6>
`;
